pkg/httpapi/api.go

```diff
@@ -287,12 +295,71 @@ func (a *APIRouter) GetApplicationDetails(w http.ResponseWriter, r *http.Request
 		Revision: revision,
 	}
 
+	var appServices = make([]envHealthResource, 0)
+	var appDeployments = make([]envHealthResource, 0)
+	var appSecrets = make([]envHealthResource, 0)
+	var appRoutes = make([]envHealthResource, 0)
+	var appRoleBindings = make([]envHealthResource, 0)
+	var appClusterRoles = make([]envHealthResource, 0)
+	var appClusterRoleBindings = make([]envHealthResource, 0)
+
+	if len(app.Status.Resources) > 0 {
+		for _, aResource := range app.Status.Resources {
+			if aResource.Kind == kindService {
```

Comment:
I would do a "switch" statement instead of a series of else-ifs.

Reply:
Converted to use "switch"